saleor/payment/gateways/adyen/plugin.py

```diff
@@ -229,9 +235,16 @@ def get_payment_gateway_for_checkout(
     def process_payment(
         self, payment_information: "PaymentData", previous_value
     ) -> "GatewayResponse":
+        params = urlencode({"payment": payment_information.payment_id})
```

Comment:
It is not safe to build URL with the only payment_id. It is base64 which is easy to decode. We should add here checkout_id also